dev/com.ibm.ws.jdbc/src/com/ibm/ws/jdbc/DataSourceService.java

```diff
@@ -557,14 +559,14 @@ protected void init() throws Exception {
         final boolean trace = TraceComponent.isAnyTracingEnabled();
 
         try {
-            PropertyService vProps = new PropertyService();
-            NavigableMap<String, Object> wProps = parseConfiguration(properties, vProps);
+            PropertyService vendorProps = new PropertyService();
+            NavigableMap<String, Object> wasProps = parseConfiguration(properties, vendorProps);
 
             // Clone properties so that we can later detect modifications from the current values
-            vProps = (PropertyService) vProps.clone();
+            vendorProps = (PropertyService) vendorProps.clone();
 
-            String jndiName = (String) wProps.remove(JNDI_NAME);
-            String type = (String) wProps.remove(DSConfig.TYPE);
+            String jndiName = (String) wasProps.remove(JNDI_NAME);
+            String type = (String) wasProps.remove(DSConfig.TYPE);
```

Comment:
I've looked at this class a few times over the past few months and `vProps` and `wProps` always confused me, so I wanted to take this opportunity to give them more clear names. Instead of `wasProps` I'll do `libertyProps`

Reply:
If you do change it, please make sure it is done consistently.  In the current set of changes, wProps is sometimes left in place and sometimes changed, and other fields that are named after it, such as WPROPS_TO_SKIP, aren't changed.  The most confusing code would be code that intermixes between the two.